pyproject.toml

```diff
@@ -166,3 +166,4 @@ markers =[
     "agentjobs",
     "unit",
 ]
+log_level = "info"
```

Comment:
Updating the default log level for `pytest` to `info`.
Makes failures easier to debug.